.github/workflows/ci-release-label.yml

```diff
@@ -0,0 +1,90 @@
+name: ci-release-label
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+
+jobs:
+  compile:
```

Comment:
we shouldn't need all four of these jobs. We just one 1 job, which will call a _new_ script for end to end testing. We could leave that script blank for now